test/src/server/helpers/revisions.js

```diff
@@ -0,0 +1,57 @@
+import * as revisions from '../../../../src/server/helpers/revisions';
+import {createMultipleRevisions, truncateEntities} from '../../../test-helpers/create-entities';
+import chai from 'chai';
+import orm from '../../../bookbrainz-data';
+
+
+const {expect} = chai;
+chai.use(require('chai-sorted'));
+
+
+describe('getOrderedRevisions', () => {
+	before(async () => {
+		await createMultipleRevisions(1000);
+	});
+	after(truncateEntities);
+
+	it('ordered revisions should be sorted and have the required keys', async () => {
+		const from = 0;
+		const size = 1000;
+		const orderedRevisions = await revisions.getOrderedRevisions(from, size, orm);
+		orderedRevisions.forEach(revision => {
+			expect(revision).to.have.keys(
+				'authorId',
+				'createdAt',
+				'editor',
+				'entities',
+				'revisionId'
+			);
+		});
+		expect(orderedRevisions).to.be.descendingBy('createdAt');
+	});
+
+	it('offset, when having higher value, should be working properly ', async () => {
```

Comment:
"it should return the expected subset of revisions when passed an offset (from)"